nasa/openmct
# ESLint rules: array-callback-return, no-invalid-this, func-style

Added the following rules to ESLint config and resolved issues found:

"array-callback-return": "error",
"no-invalid-this": "error",
"func-style": ["error", "declaration"]

Partially addresses: https://github.com/nasa/openmct/issues/3059 (along with https://github.com/nasa/openmct/pull/3139)

**Testing instructions**

1. Make code changes violating above rules.
2. Run npm run lint.
3. See that violations of rules are observed.
4. Undo changes and see that violations are gone.
5. (note: violations should also be observed in editor)

**Author Checklist**

1. Changes address original issue? Yes
2. Unit tests included and/or updated with changes? N/A
3. Command line build passes? Yes
4. Changes have been smoke-tested? Yes
5. Testing instructions included? Yes
